### DISCUSSION TITLE
Adding pomodoro timer to leechblock

### DISCUSSION THREAD

**Discussion by cenkozan** [proginosko/LeechBlockNG#497] [Ideas]
I know you know about pomodoro. I had terrible procrastination problems before I discovered the tomato! But it helped 100% with my smart phone addled brain to get into the "I'm doing real work" mindset. So we could just add a pomodoro option to set work time like 25 minutes, and 5 minutes of rest - on top of the hours in a days that leechblock is limiting. Somehow to let us back into the blocked sites for the rest period. I'm thinking we will also need "start work" "stop work" buttons somewhere in the UI - but I'm not sure how best to implement those...

Hope this is making sense.

Many thanks again and again for this AWESOME extension!!!

**Comment by proginosko** [comment]
Wouldn't the time limit option have the same effect? Set it to allow 5 minutes in every 30 minutes.

**Reply by cenkozan** [reply]
Many many thanks. I'll do that. Issue is resolved quickly but hopefully others will be able to find this solution too!
Cheers!
